packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/example.html

```diff
@@ -456,6 +456,27 @@ <h3>Loading a filterable list</h3>
   <gux-dropdown-multi id="loading-2" filter-type="starts-with" loading>
     <gux-listbox-multi aria-label="Animals"> </gux-listbox-multi>
   </gux-dropdown-multi>
+
+  <h2>Grouped Options</h2>
+  <gux-dropdown-multi>
+    <gux-listbox-multi aria-label="Animals">
+      <gux-option-group>
```

Comment:
The native HTML element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) this mimics uses a `label` property. Usually I prefer the slot approach but maybe in this case we should copy the native one. At the very least I think we should change the slot name to `label`.